worker/firewaller/firewaller.go

```diff
@@ -573,7 +573,13 @@ func (fw *Firewaller) reconcileInstances() error {
 			return err
 		}
 		machineId := machined.tag.Id()
-		initialRules, err := instances[0].IngressRules(machineId)
+
+		fwInstance, ok := instances[0].(instance.InstanceFirewaller)
+		if !ok {
+			return errors.Errorf("firewaller using unexpected instance type")
```

Comment:
why error? shouldn't we just be ignoring the providers/instances that don't implement firewalling?